src/Components/common/LoadingState/Error/index.js

```diff
@@ -0,0 +1,21 @@
+import {
```

Comment:
Nie wiem jak dodawać komentarze do ikonek :) dlatego tutaj wpisze - jeżeli tworzysz plik z obrazkami, to powinien być z małej litery, bo jeżeli jest z dużej to jest to mylące, bo z dużej litery tworzymy komponenty. Dodatkowo uwazam ze powinnismy utworzyc jeden folder z obrazkami i wszyscy z tego zaciagac do komponentów, bo takie porozrzucane obrazki tez średnio wyglądaja.